docs/AuthenticationMethods.md

```diff
@@ -99,9 +105,9 @@ df.write
   .option(KustoOptions., "MyTable")
   .save()
 ```
-## Device authentication
-If no authentication parameters were passed to the connector - the connector will write a token to the console, 
-which can be used to authenticate at https://login.microsoftonline.com/common/oauth2/deviceauth and will allow
- temporary access. 
+## Device Authentication
+If no authentication parameters were passed. The connector will request for user authentication by writing a token 
+to the console, this token can be used to authenticate at https://login.microsoftonline.com/common/oauth2/deviceauth 
+and will allow temporary access. 
 
 **Note:** This method is not recommended!   
```

Comment:
Please elaborate why, and when it is reasonable to use it (I guess not for production but OK for ad-hock access e.g. for experimentation)